carbon-design-system/carbon-addons-iot-react
# fix(hotspoteditortooltiptab): fix for selected color not showing

Closes #2401

**Summary**
Implemented fix Selected color not showing in hotspoteditortooltiptab story

**Change List (commits, features, bugs, etc)**
Added hotspotIconFillColors to storybook examples and made the prop required

**Acceptance Test (how to verify the PR)**
1. Go to https://deploy-preview-2402--carbon-addons-iot-react.netlify.app?path=/story/2-watson-iot-experimental-☢%EF%B8%8F-hotspoteditormodal-hotspoteditortooltiptab--with-state-in-story
2. Select a color and verify that it is displayed

**Regression Test (how to make sure this PR doesn't break old functionality)**
Verify that other stories for hotspoteditortooltiptab works as expected
